src/app/clusters/scenes-server/scenes-server.cpp

```diff
@@ -762,8 +773,7 @@ void ScenesServer::HandleCopyScene(HandlerContext & ctx, const Commands::CopySce
         AddResponseOnError(ctx, response, Attributes::SceneTableSize::Get(ctx.mRequestPath.mEndpointId, &endpointTableSize)));
 
     // Get Scene Table Instance
-    scenes::SceneTable<scenes::ExtensionFieldSetsImpl> * sceneTable =
-        scenes::GetSceneTableImpl(ctx.mRequestPath.mEndpointId, endpointTableSize);
+    SceneTable * sceneTable = scenes::GetSceneTableImpl(ctx.mRequestPath.mEndpointId, endpointTableSize);
```

Comment:
All these cleanup bits should _also_ not be in the same PR as substantive changes.

Reply:
Pr created [here](https://github.com/project-chip/connectedhomeip/pull/27079)